performance-benchmarks/README.md

```diff
@@ -0,0 +1,9 @@
+# Ballerina-performance-benchmarks
```

Comment:
Shall we rename the module as benchmarks? The word performance is redundant. See also https://en.wikipedia.org/wiki/Benchmark_(computing)

Reply:
+1

Reply:
fixed